tools/FakeItEasy.Build/Program.cs

```diff
@@ -63,6 +64,16 @@ public static void Main(string[] args)
                 forEach: ProjectsToPack,
                 action: project => Run("dotnet", $"pack {project.Path} --configuration Release --no-build --output {Path.GetFullPath("artifacts/output")}"));
 
+            Target(
+                "accept-api",
+                () =>
+                    {
+                        foreach (var received in Directory.EnumerateFiles("tests/FakeItEasy.Tests.Approval", "*.received.txt", SearchOption.AllDirectories))
+                        {
+                            File.Copy(received, received.Replace(".received.txt", ".approved.txt"), overwrite: true);
+                        }
+                    });

```

Comment:
How do we use this? It assumes the .received files have already been generated... So, run `approve`, which will fail, then run `accept-api`?

Reply:
Maybe we should add some kind of flag (e.g. an environment variable) to change the behavior of the approval tests. If the flag is set, don't compare received against approved, but just copy received to approved?

Reply:
> So, run approve, which will fail, then run accept-api?

I usually build everything, find the failure on `approve` and then manually copy. I was just trying to ease the last step. Do you do something different?

> Maybe we should add some kind of flag (e.g. an environment variable) to change the behavior of the approval tests. If the flag is set, don't compare received against approved, but just copy received to approved?

Could do, I suppose. I guarantee you 100% I will forget.

Reply:
> I usually build everything, find the failure on `approve` and then manually copy. I was just trying to ease the last step. Do you do something different?

For me, each test that fail cause my diff tool (Devart Code Compare) to open the received and approved files, then I just need hit the "Copy all changes to right" button. Doesn't it do that for you?

> Could do, I suppose. I guarantee you 100% I will forget.

Yes, I'd probably forget too. Your way is fine, I think

Reply:
For me, it pops up copies of... something. WinMerge? I dunno. Then I panic and close them all and copy over! I could use the differ to copy.